boards/nxp/frdm_ke15z/doc/index.rst

```diff
@@ -0,0 +1,229 @@
+.. _frdm_ke15z:
+
+NXP FRDM-KE15Z
+#############
+
+Overview
+********
+
+The FRDM-KE15Z is a development board for NXP Kinetis KE1xZ 32-bit
+MCU-based platforms. The FRDM-KE15Z contains a robust TSI module
+with up to 50 channels which makes this board highly flexible
+for touch keys. Offers options for serial
+communication, flash programming, and run-control debugging.
+
+.. figure:: frdm_ke15z.webp
+   :align: center
+   :alt: FRDM-KE15Z
+
+   FRDM-KE15Z (Credit: NXP)
+
+Hardware
+********
+
+- MKE15Z256VLL7 MCU (up to 72 MHz, 512 KB flash memory, 96 KB RAM)
+- I/O headers for easy access to MCU I/O pins
+- OpenSDA Debug Circuit with a virtual serial port
+- NXP sensor, FXOS8700CQ
+- Touch electrodes in the self-capacitive mode
+- Supported by MCUXpresso SDK, IDE, and Config Tools
+- Compatible with FRDM-TOUCH, FRDM-MC-LVBLDC, and Arduino® boards
+- User Components such as Reset; RGB LED and four user buttons
```

Comment:
I understand it says this on the marketing page, but there does not appear to be four user buttons

Reply:
Right I missed this one.